RecoHI/Configuration/python/Reconstruction_hiSignal_cff.py

```diff
@@ -12,7 +12,7 @@
 #Tracks
 hiSignalGlobalPrimTracks = hiGlobalPrimTracks.clone()
 hiSignalSelectedTracks = hiSelectedTracks.clone()
-hiSelectedTracks.src = cms.InputTag("hiSignalGlobalPrimTracks")
+hiSelectedTracks.src = "hiSignalGlobalPrimTracks"
```

Comment:
I imagine that the original intention was to modify the cloned `hiSignalSelectedTracks,` and then include in the Task, instead of the `hiSelectedTracks`